foods.html

```diff
@@ -5,7 +5,27 @@
   <title>Quantified Self - Manage Foods</title>
 </head>
 <body>
+  <h2>Manage Foods</h2>
 
+  <form class="food-form">
+    Name:<br>
+    <input type="text" name="name" value=""><br>
+    Calories:<br>
+    <input type="number" name="calories" value=""><br>
+    <input type="submit" name="" value="Add Food">
+  </form>
+
+  <form class="filter">
+    Filter by Name:<br>
+    <input type="text" name="filter" value="">
+  </form>
+
+  <table class="foods-table">
+    <th>Name</th>
+    <th>Calories</th>
+  </table>
+  <script src='https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js'>
+  </script>
```

Comment:
@corywest @Tyjoo26 @maxscores
Is this how we are supposed to require jquery, even though we used `npm install jquery` ? We weren't able to get it to work otherwise and this worked.